content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,180 @@
+=======================================================
+Connecting the Windows IoT software to an Odoo database
+=======================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
+computer. This requires a Windows operating system with an Odoo 16 or later database.
+
+The Windows Virtual :abbr:`IoT (Internet of Things)` box works the same way as a physical :abbr:`IoT
+(Internet of Things)` box, with the ability to run most of the same devices. All :abbr:`POS (Point
+of Sale)` devices work with it, such as a scale or printer. Payment terminals will also work, but it
+should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not compatible. *These
+include cameras or measurement tools.*
+
+Pre-requisites
+==============
+
+The following items will be needed to complete the Windows IoT installation.
+
+- Odoo 16 database or any version above.
+- :abbr:`IoT (Internet of Things)` compatible devices (except those mentioned above). Refer to:
+  `Odoo's compatible IoT devices <https://www.odoo.com/app/iot-hardware>`_.
+- Device drivers for Windows.
+
+  .. note::
+     Odoo recommends using an updated, recent version of Windows as some older operating systems can
+     cause the Windows virtual :abbr:`IoT (Internet of Things)` to not work.
+
+- Windows computer (laptop, desktop, server).
+- Odoo :abbr:`IoT (Internet of Things)` subscription. Refer to: :ref:`iot/iot-eligibility`.
+
+Connect the Windows virtual Iot box to an Odoo database
+=======================================================
+
+The Windows virtual IoT box is simple to setup in just a few easy steps. Follow this process when
+installing the Windows virtual IoT software for the first time.
+
+Download and initial installation
+---------------------------------
+
+To begin the installation, navigate to the Odoo 16 or higher installation package for Enterprise or
+Community Windows at `Odoo's Download Page <https://odoo.com/download>`_. Next, Install and setup
+the Odoo `.exe` file. After the instructions screen, click :guilabel:`Next` to start the
```

Comment:
Totally optional - Im a fan of using the :file: tag for file extentions and filenames :)
```suggestion
the Odoo :file:`.exe` file. After the instructions screen, click :guilabel:`Next` to start the
```